addons/account/views/account_move_views.xml

```diff
@@ -606,7 +606,13 @@
                                        options='{"always_reload": True, "no_quick_create": True}'
                                        attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt'))]}"/>
 
-                                <field name="ref"/>
+                                <div class="o_td_label">
+                                    <label for="ref"
+                                           attrs="{'invisible':[('move_type', '=', 'in_invoice')]}" />
```

Comment:
`in', ('in_invoice', 'in_receipt', 'in_refund')`

Reply:
done